whylog_vim/proxy/teacher/__init__.py

```diff
@@ -1,17 +1,60 @@
+from whylog_vim.consts import EditorStates, Messages
+from whylog_vim.output_formater.teacher_formater import TeacherFormater
+from whylog_vim.proxy.teacher.utils import get_next_parser_id
+from whylog_vim.proxy.teacher.exceptions import CannotGoToPosition
+
+
 class TeacherProxy(object):
     def __init__(self, teacher, editor, main_proxy):
         self.teacher = teacher
         self.editor = editor
         self.main_proxy = main_proxy
 
+    def new_lesson(self):
+        front_input = self.editor.get_front_input()
+        self.teacher.add_line(get_next_parser_id(), front_input, effect=True)
+        self.origin_file_name = self.editor.get_current_filename()
+        print(Messages.ADDED_EFFECT)
```

Comment:
don't you need to `from __future__ import print_statement` to use it under python2?